scripts/ci/travis/steps.sh

```diff
@@ -83,6 +83,13 @@ step_install_chainermn_test_deps() {
 }
 
 
+step_before_install_chainerx_test_deps() {
+    if [[ $TRAVIS_OS_NAME = "linux" ]]; then
```

Comment:
It needs to be mentioned that separate installation is not needed because OSX has LAPACK installed natively with [Accelerate framework](https://developer.apple.com/documentation/accelerate).
I tested OSX Travis build in my repo, it indeed works ([link](https://travis-ci.org/IvanYashchuk/chainer/jobs/559871479#L2250)). And CPU linalg tests are [passed](https://travis-ci.org/IvanYashchuk/chainer/jobs/559871479#L4368).